src/Migrations/Version20180711070226.php

```diff
@@ -14,37 +14,65 @@
 namespace Sylius\InvoicingPlugin\Migrations;
 
 use Doctrine\DBAL\Schema\Schema;
-use Doctrine\Migrations\AbstractMigration;
+use Sylius\Bundle\CoreBundle\Doctrine\Migrations\AbstractPostgreSQLMigration;
 
-/**
- * Auto-generated Migration: Please modify to your needs!
- */
-final class Version20180711070226 extends AbstractMigration
+final class Version20180711070226 extends AbstractPostgreSQLMigration
```

Comment:
Why did you use and edit an existing migration? As this is an initialization migration for PostgreSQL, just create a new one.